### PR TITLE
Fix h1 on brandlist and supplierlist module that shouldn't be an h1

### LINKED ISSUE #18409: The ps_brandlist has a big seo error - the block title has H1 instead of a simple link with h6 calss
#### Describe the bug

Hi
I've seen that the title of `ps_brandlist` module has `H1` instead of a simple link with `h6` class

it is necessary to fix it to avoid SEO problems with the main `H1`

The issue previously reported [here](http://forge.prestashop.com/browse/BOOM-3297).

#### Steps to Reproduce

1. Install the `ps_brandlist` module
2. Go to the FO => Categories page
3. Scroll down to ps_brandlist block
4. Inspect element
5. See error: the title of `ps_brandlist` module has `H1` instead of a simple link with `h6` class

**Screenshots**
image

#### Additional information

* PrestaShop version: 1.7.5.2, 1.7.6.4, 1.7.7.x
* PHP version: 7.2
* BrandList v1.0.2

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | There was an H1 on brandlist and supplierlist module on category page filters that shouldnt be an h1
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #18409.
| How to test?  | Install the ps_brandlist module, Go to the FO => Categories page, Scroll down to ps_brandlist block, Inspect element, See error: the title of ps_brandlist module has H1 instead of a simple link with h6 class

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18459)

### CODE REVIEW

File: themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl [hunk 1]
```diff
@@ -25,11 +25,16 @@
...
-        {l s='Brands' d='Shop.Theme.Catalog'}
-      {if $display_link_brand}</a>{/if}
-    </h1>
+    {if $display_link_brand}
+      <a href="{$page_link}" class="h6 text-uppercase facet-label" title="{l s='brands' d='shop.theme.catalog'}">
+        {l s='brands' d='shop.theme.catalog'}
+      </a>
+    {else}
+      <p class="h6 text-uppercase facet-label">
+        {l s='brands' d='shop.theme.catalog'}
```
Comment (matks) [inline themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl]:
```suggestion
        {l s='Brands' d='shop.theme.catalog'}
```
It seems you changed that

File: themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl [hunk 2]
```diff
@@ -25,11 +25,16 @@
...
 <div id="search_filters_brands">
   <section class="facet">
-    <h1 class="h6 text-uppercase facet-label">
-      {if $display_link_brand}<a href="{$page_link}" title="{l s='Brands' d='Shop.Theme.Catalog'}">{/if}
-        {l s='Brands' d='Shop.Theme.Catalog'}
-      {if $display_link_brand}</a>{/if}
-    </h1>
+    {if $display_link_brand}
+      <a href="{$page_link}" class="h6 text-uppercase facet-label" title="{l s='brands' d='shop.theme.catalog'}">
+        {l s='brands' d='shop.theme.catalog'}
```
Comment (matks) [inline themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl]:
```suggestion
        {l s='Brands' d='shop.theme.catalog'}
```

File: themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl (hunk 1 above)
Comment (NeOMakinG) [reply themes/classic/modules/ps_brandlist/views/templates/hook/ps_brandlist.tpl]: Damn, what happened, it was a copy pasta

**Review by matthieu-rolland — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is OK with `ps_brandlist` module and `ps_supplierlist`.
1. with link
image
2. with no link
image

Thanks!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**